README.md

```diff
@@ -12,96 +12,96 @@ The OpenTelemetry Operator is an implementation of a [Kubernetes Operator](https
-
-```yaml
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: my-java-app
-spec:
-  template:
-    metadata:
-      annotations:
-        otel.splunk.com/inject-java: "true"
-    spec:
-      containers:
-      - name: my-java-app
-        image: my-java-app:latest
+kubectl get pods -n cert-manager
+NAME                                       READY   STATUS    RESTARTS   AGE
+cert-manager-7c9c58cbcb-jwwkk              1/1     Running   0          5m1s
+cert-manager-cainjector-5d88544c9c-chwhr   1/1     Running   0          5m1s
+cert-manager-webhook-85f88ffb5b-4hrpb      1/1     Running   0          5m1s
+kubectl get pods -n splunk-otel-operator-system
+NAME                                                       READY   STATUS    RESTARTS   AGE
+splunk-otel-agent-pp8wn                                    1/1     Running   0          68s
+splunk-otel-cluster-receiver-8f666b5b8-wbncp               1/1     Running   0          68s
+splunk-otel-operator-controller-manager-67b86fcf5c-f2sqq   1/1     Running   0          3m38s
 ```
 
-This will automatically inject [Splunk OpenTelemetry Java Agent](github.com/signalfx/splunk-otel-java) into the pod and configure it to send telemetry to the OpenTelemetry agents deployed by the operator.
-
-Right now the following annotations are supported:
-
-### otel.splunk.com/inject-java
-
-When this instrumentation is set to `"true"` on a pod, the operator automatically instruments the pod with the Splunk OpenTelemetry Java agent and configures it to send all telemetry data to the OpenTelemetry agents managed by the operator. 
+**_WARNING:_** Until the OpenTelemetry Collector format is stable, changes may be required in the above example to remain  
+compatible with the latest version of the Splunk OpenTelemetry Operator and Splunk OpenTelemetry Collector.  
+  
+## Automatically instrumenting k8s pods  
+  
+This operator can automatically inject configuration and instrumentation agents into Kubernetes pods on demand. In order to do so, you'll need to annotate the pods you want to instrument or auto-configure. For example, if your deployment looks like the following:  
+  
+```yaml  
+apiVersion: apps/v1  
+kind: Deployment  
+metadata:  
+  name: my-java-app  
+spec:  
+  template:  
+    spec:  
+      containers:  
+      - name: my-java-app  
+        image: my-java-app:latest  
+```
 
-### otel.splunk.com/inject-config
+Then you can automatically instrument it by adding the `otel.splunk.com/inject-java: "true"` annotation to the Pod spec (not the deployment):
 
-When this instrumentation is set to `"true"` on a pod, the operator only configures the pod to send all telemetry data to the OpenTelemetry agents managed by the operator. Pods are not instrumented in this case and that is left to the user.
+```
+kubectl patch deployment my-java-app -p '{"spec": {"template":{"metadata":{"annotations":{"otel.splunk.com/inject-java":"true"}}}} }' --namespace my-java-app-namespace
```

Comment:
Are we sure we want to recommend this? This mean the change will not be committed into customers version control system and could be lost on subsequent deploys.

Reply:
Good point. Since we show how to do this in the example I'm adding, I'm revert this section.